src/main/java/duke/Event.java

```diff
@@ -0,0 +1,49 @@
+package duke;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+
+    private String at;
+    private LocalDate start_date;
```

Comment:
Variable names must be in camelCase. (i.e. startDate) This applies to the variables on the next few lines as well. 😃